### PR TITLE
Added theme

### PR DESCRIPTION
**Pull request by HugoValentine**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "name": "hyper-rose",
+    "description": "A theme based on bright pink colours",
+    "type": "theme",
+    "preview": "https://raw.githubusercontent.com/HugoValentine/hyper-rose/master/screenshot.png",
```
Comment (vassbence) [inline plugins.json:6]:
```suggestion
```

**Comment by vassbence** [comment]
Sorry for the ton of changes, we have updated the [plugin submission guidelines](https://hyper.is/store/submit), could you please merge master one last time and also add your preview image with the name of the plugin to `/public/store`?

**Comment by HugoValentine** [comment]
> Sorry for the ton of changes, the [plugin submission guidelines](https://hyper.is/store/submit) have been updated recently, could you please merge master one last time and also add your preview image with the name of the plugin to `/public/store`?

No problem, mate.

**Comment by vassbence** [comment]
Closing this, continuing in #192

**Pull request closed without merging**